traffic_ops/v3-client/deliveryservice.go

```diff
@@ -206,11 +206,13 @@ func (to *Session) GetDeliveryServicesByCDNID(cdnID int) ([]tc.DeliveryServiceNu
 	return to.GetDeliveryServicesByCDNIDWithHdr(cdnID, nil)
 }
 
-func (to *Session) GetDeliveryServiceNullableWithHdr(id string, header http.Header) (*tc.DeliveryServiceNullable, ReqInf, error) {
+// GetDeliveryServiceNullableWithHdr fetches the Delivery Service with the given ID.
+func (to *Session) GetDeliveryServiceNullableWithHdr(id int, header http.Header) (*tc.DeliveryServiceNullableV30, ReqInf, error) {
```

Comment:
Yeah, this'll need to be reverted if this doesn't get into 5.0.0. Which kind of sucks, but I can't put it in the 4.0 client because that doesn't exist yet, so maybe I should just not change this at all?

Reply:
Yeah, we should probably stop backporting new features into 5.0.x. It's confusing as a developer to not know what release a change is going into, when there are only certain things you can do or have to do around a release boundary.